lib/CL/devices/common.c

```diff
@@ -1769,7 +1769,11 @@ pocl_init_default_device_infos (cl_device_id dev,
   dev->kernellib_name = strdup(kernellib);
   if (dev->kernellib_subdir == NULL)
     dev->kernellib_subdir = "host";
-  dev->llvm_abi = pocl_get_llvm_cpu_abi ();
+  dev->llvm_abi = pocl_get_llvm_cpu_abi();
```

Comment:
Check the C style which is GNU. The clang-format-diff.sh script should fix the basic formatting for you.